composer.json

```diff
@@ -58,7 +58,7 @@
     "ext-curl": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.5.*",
+    "phpunit/phpunit": "4.8.*",
```

Comment:
@christophermancini this should probably be part of a different PR (just being picky as usual)